roles/kubevirt/tasks/provision.yml

```diff
@@ -90,54 +92,41 @@
       when: deploy_demo == true
 
 # VM Templates manifests
-- name: Check for vm templates in {{ kubevirt_template_dir }}
-  stat:
-    path: "{{ kubevirt_template_dir }}/{{ item }}.yaml"
-  with_items:
-    - "{{ default_vm_templates }}"
-  register: byo_vm_templates
-
-- name: Copy VM templates to /tmp
-  copy:
-    src: "{{ kubevirt_template_dir }}/{{ item }}.yaml"
-    dest: "/tmp/{{ item }}.yaml"
-  with_items: "{{ byo_vm_templates.results | selectattr('stat.exists') | map(attribute='item') | list }}"
-  when: "{{ byo_vm_templates.results | selectattr('stat.exists') | map(attribute='item') | list | length > 0 }}"
-
-- name: Check for vm templates in {{ offline_template_dir }}
-  stat:
-    path: "{{ offline_template_dir }}/v{{ version }}/{{ item }}.yaml"
-  with_items:
-    - "{{ default_vm_templates }}"
-  register: offline_vm_templates
-
-- name: Copy VM templates to /tmp
-  copy:
-    src: "{{ offline_template_dir }}/v{{ version }}/{{ item }}.yaml"
-    dest: "/tmp/{{ item }}.yaml"
-  with_items: "{{ offline_vm_templates.results | selectattr('stat.exists') | map(attribute='item') | list }}"
-  when: "{{ offline_vm_templates.results | selectattr('stat.exists') | map(attribute='item') | list | length > 0 }}"
-
-- name: Set the default VM templates URL
-  set_fact:
-    templates_url: "https://raw.githubusercontent.com/kubevirt/kubevirt/v{{ version }}/cluster/examples"
-  when: "{{ '0.6.0' is version(version, '<') }}"
-
-- name: Set the default VM templates URL
-  set_fact:
-    templates_url: "https://raw.githubusercontent.com/kubevirt/kubevirt/v{{ version }}/cluster"
-  when: "{{ '0.6.0' is version(version, '>=') }}"
-
-- name: Download KubeVirt default VM templates
-  get_url:
-    url: "{{ templates_url }}/{{ item }}.yaml"
-    dest: "/tmp/{{ item }}.yaml"
-  with_items:
-    - "{{ default_vm_templates }}"
-  when: platform == "openshift" and "{{ byo_vm_templates.results | selectattr('stat.exists') | map(attribute='item') | list | length == 0 }}" and "{{ offline_vm_templates.results | selectattr('stat.exists') | map(attribute='item') | list | length == 0 }}"
-
-- name: Create default VM templates in OpenShift Namespace
-  shell: "oc apply -f /tmp/{{ item }}.yaml -n openshift"
-  with_items:
-    - "{{ default_vm_templates }}"
+- name: install vm templates manifests
+  block:
+  - name: Check for vm templates in {{ kubevirt_template_dir }}
+    stat:
+      path: "{{ kubevirt_template_dir }}/common-templates.yaml"
```

Comment:
this comes from kubevirt-templates rpm.

Reply:
hm it just strange that `kubevirt.j2` and `common-templates.yaml` placed under the same directory I was expected that the will be under different directories

Reply:
Before this patch it was the same.
```yaml
- name: Check for vm templates in {{ kubevirt_template_dir }}
  stat:
    path: "{{ kubevirt_template_dir }}/{{ item }}.yaml"
  with_items:	 
    - "{{ default_vm_templates }}"
```

I just stop using the templates because they don't work (image point to registry5000 and tag to devel)